src/transactions/repository_test.go

```diff
@@ -0,0 +1,105 @@
+package transactions
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/lungria/mono"
+	"go.mongodb.org/mongo-driver/bson/primitive"
+)
+
+var txn1 = Transaction{
+	ID:        primitive.NewObjectID(),
+	AccountID: "fCfTvZgMihvz_URzZSqVEf",
+	StatementItem: mono.StatementItem{
```

Comment:
I don't think this is the best approach.
https://www.mongodb.com/blog/post/6-rules-of-thumb-for-mongodb-schema-design-part-1
mongoDb docs suggest to store data in the same way you use it.
I suggest that basic structure of the model of all our transactions must be *exactly* the same as we use on `GET /transactions` endpoint:
```
	ID   
	CategoryId // or maybe embed full category object? not sure
	AccountId
        DateTime // In UTC, based on when transaction really occured - https://stackoverflow.com/a/15952652/6878401 - ISO 8601 for JSON and mongodb Date type https://docs.mongodb.com/manual/reference/bson-types/#document-bson-type-date 
        Amount // In UAH. I suggest to postpone multi-currency support. we still can store this info in BankTransaction
        Description
```
We may, if we need to store monobank transaction as an embedded field, as it done here.
I suggest  rename it to BankTransaction or something like that - there is 85% chance that this app would also support PrivatBank transactions. They  don't support webhooks at the moment, but we can export CSV and parse it